lib/Github/Model/Fork.php

```diff
@@ -0,0 +1,381 @@
+<?php
+namespace Github\Model;
+
+use function Makasim\Values\get_object;
+use function Makasim\Values\get_value;
+
+class Fork
```

Comment:
```suggestion
final class Fork
```

should be final, there is only one implementation and no one should extend it IMO